python/setup.py

```diff
@@ -141,8 +147,8 @@ def build_extensions(self):
 
 ext_modules = [
 	Extension(
-		'SBCK.tools.__tools_cpp',
-		['SBCK/tools/src/tools.cpp'],
+		os.path.join(here, 'SBCK.tools.__tools_cpp'),
+		[ os.path.join(here, 'SBCK/tools/src/tools.cpp') ],
```

Comment:
@Zeitsperre  I understand the second line, but I'm not sure about the first? 
https://setuptools.pypa.io/en/latest/userguide/ext_modules.html?highlight=Extension#extension-api-reference
I think the `name` argument should the  dotted name of the modue. The current directory doesn't seem to matter?